superset/cli.py

```diff
@@ -628,3 +628,18 @@ def alert() -> None:
             resolution=6000,
             session=session,
         )
+
+
+@superset.command()
+@with_appcontext
+def create_db() -> None:
+    """Resets database"""
+    db.create_all()
+
+
+@superset.command()
+@with_appcontext
+@click.confirmation_option(prompt="Are you sure you want to drop the db?")
```

Comment:
one other suggestion here. I think it could be helpful to confirm the name of the database that we are dropping since it's only the config that switches this from dev to test db.